src/components/Weapons.vue

```diff
@@ -155,6 +134,11 @@ export default {
             background: $yellow;
             color: black;
           }
+
+          &.required-weapon {
+            border-left: 5px solid $purple;
+            padding: 25px 25px 25px 20px; // to maintain text center alignment with the extra thick border
```

Comment:
```suggestion
            padding: 25px 25px 25px 20px; // to maintain text center alignment with the extra thick border
            
            @media (max-width: $tablet) {
              padding: 15% 15% 15% calc(15% - 5px) // to maintain text center alignment with the extra thick border
            }
```
Don't forget the padding change on mobile/tablet.